amr-wind/equation_systems/icns/source_terms/DragForcing.H

```diff
@@ -13,6 +13,25 @@ namespace amr_wind::pde::icns {
  *
  *
  */
+
```

Comment:
since this is standalone and just being used here, can you enclose is in `namespace{}` ? read up on anonymous namespaces if you are curious

Reply:
Actually it looks like you are doing this already. Move this to the relevant section in `DragForcing.cpp`

Reply:
And while you are at it, change all those decorators to use `AMREX_GPU_DEVICE` as well ;)